.github/workflows/release.yaml

```diff
@@ -8,21 +8,33 @@ on:
       - .github/renovate.json5
       - pyproject.toml
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
 jobs:
+  get-version:
+    uses: ./.github/workflows/get-version.yaml
+
   ci:
     name: Build & Test
     uses: ./.github/workflows/build-and-test.yaml
     secrets: inherit
+    needs:
+      - get-version
+    with:
+      snap_channel: ${{ needs.get-version.outputs.version }}/edge
 
   release:
     name: Release charm
     needs:
       - ci
+      - get-version
     uses: canonical/data-platform-workflows/.github/workflows/release_charm.yaml@v13.0.0
     with:
       # TODO: change this when we actually want to cutover the charm to main
       # channels
-      channel: 1.21-ops/edge
+      channel: ${{ jobs.get-version.outputs.version }}-ops/edge
```

Comment:
The version value is picked from `version` file and is plain `1.2X`

Reply:
Sorry, I didn't make it clear. 
What I mean was that since you added
```
    needs:
      - ci
+    - get-version
```
Can this be 
```
channel: ${{ needs.get-version.outputs.version }}-ops/edge
```
for consistency just as you did for the `ci` job or drop the `needs` and use the above version?